src/displayapp/screens/Steps.cpp

```diff
@@ -28,6 +28,8 @@ Steps::Steps(Pinetime::Applications::DisplayApp* app,
 
   stepsCount = motionController.NbSteps();
   currentTripSteps = stepsCount - motionController.GetTripSteps();
```

Comment:
I think this should just be `motionController.GetTripSteps()`, not `stepsCount - motionController.GetTripSteps()`.
```suggestion
  currentTripSteps = motionController.GetTripSteps();
```

Reply:
The purpose of this (I believe) is to ensure steps are tracked efficiently in the time allocated. Let me explain.

Steps: 2000
I'm resetting my trip meter, so my steps are 0.
I walk 1000 steps.
Steps: 3000
Trip: 1000

I think this takes the current step count and ensures the trip is the portion of those steps correctly.

Reply:
I think that that calculation should already be done in `motionController.GetTripSteps()`, as in the `Refresh()` function, `currentTripSteps` is just set the way I suggested.